pkg/agent/storage/keyvalue_store_bbolt_test.go

```diff
@@ -1,10 +1,9 @@
-package control
+package storage
```

Comment:
I think the bbolt, memory, and ci implementations should land in their own pkg directories.

Reply:
Done!